components/econet/climate/econet_climate.h

```diff
@@ -47,6 +53,11 @@ class EconetClimate : public climate::Climate, public Component, public EconetCl
   std::string mode_id_{""};
   std::string custom_preset_id_{""};
   std::string custom_fan_mode_id_{""};
+  std::string custom_fan_mode_no_schedule_id_{""};
+  std::string custom_follow_schedule_id_{""};
+  uint8_t follow_schedule_{255};
```

Comment:
Should `follow_schedule_` actually be a bool instead of an int? It seems like it's only ever 0 or 1 in the code above.

Reply:
change it to `optional<bool> follow_schedule_`

Reply:
Updated to optional<bool> in a new commit. There are 3 states that do impact logic/functionality. The "third state" is when we haven't received the SCHEDULE datapoint yet.